lib/ui/painting.dart

```diff
@@ -1362,6 +1381,15 @@ class Path extends NativeFieldWrapperClass2 {
   Path() { _constructor(); }
   void _constructor() native 'Path_constructor';
 
+  /// Creates a new [Path] object from src path
+  /// 
+  /// This copy is fast and does not require additional memory unless the src or
```

Comment:
unless either the source path or the path returned by this constructor are modified.